Makefile

```diff
@@ -0,0 +1,5 @@
+cs:
+	php-cs-fixer fix --verbose
+
+test:
+	./vendor/bin/phpunit --coverage-text
```

Comment:
imho we do not need a makefile for running phpunit, its so straightforward. i think in this case i would also prefer the php-cs-fixer line to be in the README rather than have a Makefile. i think Makefiles are a good thing, but here its some very specific tasks only relevant when developing.